### ISSUE TITLE
Bugged tree command on CLI (duplicated box).

### ISSUE THREAD

**Issue by orl0pl** [orl0pl/box-master#1]
This is an example:
```
> tree
tree
|-- HE (0)
  |-- Third (2)
|-- 2 (1)
  |-- WTF (3)
|-- Third (2)
```

**Comment by orl0pl** [comment]
Issue seems to appears after adding box to the box with id `0`.

**Comment by orl0pl** [comment]
Issue fixed with commit 0d7e129
